backend/src/calibration.py

```diff
@@ -52,9 +51,14 @@ def __de_equ(x: np.ndarray, camsigs: np.ndarray, labref: np.ndarray) -> float:
     o = np.resize(x[18:], (6, 1))
     xyz = np.matmul(m, np.subtract(camsigs, o))
     lab = xyztolab(xyz)
+    btrgb.CalibrationResults["matrix_values"][4] = btrgb.create_matrix_value(lab[0], "CM L*_camera")
+    btrgb.CalibrationResults["matrix_values"][5] = btrgb.create_matrix_value(lab[1], "CM a*_camera")
+    btrgb.CalibrationResults["matrix_values"][6] = btrgb.create_matrix_value(lab[2], "CM b*_camera")
     xyzshape = xyz.shape[1]
     ciede = np.zeros(xyzshape)
     for i in range(0, lab.shape[1]):
         ciede[i] = ciede2000(lab[:, i], labref[:, i])
+    btrgb.CalibrationResults["matrix_values"][0] = btrgb.create_matrix_value(ciede, "CM DeltaE Values")
     err = np.mean(ciede)
+    btrgb.CalibrationResults["double_values"][0] = btrgb.create_double_value(err, "CM DeltaE Mean")
```

Comment:
Do we ever access the 'matrix_values' in other indices (specifically 1, 2, or 3)?